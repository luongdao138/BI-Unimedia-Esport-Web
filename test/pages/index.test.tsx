import React from 'react'
import renderer from 'react-test-renderer'
import HomePage from '../../pages/index'

describe('Home page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HomePage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
