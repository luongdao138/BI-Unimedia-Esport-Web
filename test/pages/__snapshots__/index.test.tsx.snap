// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
Array [
  <div
    className="parallax"
  >
    <div
      className="parallax-inner"
    />
  </div>,
  <div
    className="MuiBox-root MuiBox-root-5 makeStyles-innerWrap-1"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiBox-root MuiBox-root-6 makeStyles-parallaxContent-2"
        >
          <h2
            className="MuiTypography-root makeStyles-parallaxTitle-4 MuiTypography-h2"
          >
            common:top.title
          </h2>
          <img
            className="makeStyles-logo-3"
            src="/images/lp_exelab_logo.svg"
          />
          <button
            className="MuiButtonBase-root button-primary primary-rounded gradient primary-large"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <div
              className="circle"
            />
            <span
              className="label-primary MuiButton-label"
            >
              common:top.start_exelab
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-20 makeStyles-topContent-10"
  >
    <div
      className="MuiBox-root MuiBox-root-21 makeStyles-innerWrap-7"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-7 MuiGrid-grid-lg-7"
        >
          <div
            className="MuiBox-root MuiBox-root-22 makeStyles-leftContent-8"
          >
            <div
              className="MuiBox-root MuiBox-root-23"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-11"
                focusable="false"
                viewBox="0 0 206 43.301"
              >
                <g
                  data-name=""
                  transform="translate(0)"
                >
                  <path
                    d="M66.556,0,55.837,15.333,45.047,0H36.671L51.649,21.3,36.6,42.8h8.376L55.837,27.329,66.627,42.8H75L59.954,21.3,74.932,0Z"
                    data-name="Path 6617"
                    fill="#fff"
                    id="Path_6617"
                    transform="translate(-10.619)"
                  />
                  <path
                    d="M105.689,16.476h7.1v8.66h-7.1ZM119.105,10.3H99.3V45.864h19.805V39.688H105.689V31.241h13.487V10.3Z"
                    data-name="Path 6618"
                    fill="#fff"
                    id="Path_6618"
                    transform="translate(-28.811 -2.988)"
                  />
                  <path
                    d="M6.389,16.476h7.1v8.66h-7.1ZM19.8,10.3H0V45.864H19.8V39.688H6.389V31.241H19.876V10.3Z"
                    data-name="Path 6619"
                    fill="#fff"
                    id="Path_6619"
                    transform="translate(0 -2.988)"
                  />
                  <path
                    d="M204.569,7.67h14.339V18.388H204.569ZM197.4.5V43.446h7.169V25.558h14.339V43.446h7.17V.5Z"
                    data-name="Path 6620"
                    fill="#fff"
                    id="Path_6620"
                    transform="translate(-57.275 -0.145)"
                  />
                  <path
                    d="M257,25.558h14.339V36.277H257ZM257,7.67h10.719V18.388H257ZM249.9.5V43.446h28.607V18.388h-3.549V.5Z"
                    data-name="Path 6621"
                    fill="#fff"
                    id="Path_6621"
                    transform="translate(-72.507 -0.145)"
                  />
                  <path
                    d="M145,.5V43.446h28.678v-7.17H152.17V.5Z"
                    data-name="Path 6622"
                    fill="#fff"
                    id="Path_6622"
                    transform="translate(-42.071 -0.145)"
                  />
                </g>
              </svg>
              <h2
                className="MuiTypography-root makeStyles-iconText-12 MuiTypography-h2"
              >
                common:top.is
              </h2>
            </div>
            <h2
              className="MuiTypography-root makeStyles-topTitle-13 MuiTypography-h2"
            >
              common:top.top_title
            </h2>
            <p
              className="MuiTypography-root makeStyles-topDescription-14 MuiTypography-body1"
            >
              common:top.top_description
            </p>
            <p
              className="MuiTypography-root makeStyles-topDescriptionBottom-15 MuiTypography-body1"
            >
              common:top.download_app_version
            </p>
            <div
              className="MuiBox-root MuiBox-root-24 makeStyles-buttonWrap-16"
            >
              <button
                className="MuiButtonBase-root"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <img
                  className="makeStyles-google_app_stores-17"
                  src="/images/appstore.png"
                />
              </button>
              <div
                className="MuiBox-root MuiBox-root-25 makeStyles-buttonSeperator-18"
              />
              <button
                className="MuiButtonBase-root"
                disabled={false}
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <img
                  className="makeStyles-google_app_stores-17"
                  src="/images/googleplay.png"
                />
              </button>
            </div>
          </div>
        </div>
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-5 MuiGrid-grid-lg-5"
        >
          <div
            className="MuiBox-root MuiBox-root-26 makeStyles-rightContent-9"
          >
            <img
              className="makeStyles-rightImage-19"
              src="/images/lp_laptop.png"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="MuiBox-root MuiBox-root-34 makeStyles-bottomContent-28"
  >
    <div
      className="MuiBox-root MuiBox-root-35 makeStyles-innerWrap-27"
    >
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
      >
        <div
          className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <h2
            className="MuiTypography-root makeStyles-bottomTitle-29 MuiTypography-h2 MuiTypography-alignCenter"
          >
            common:top.recruiting_tournament
          </h2>
        </div>
        <div
          className="MuiBox-root MuiBox-root-36 makeStyles-bottomInnerContent-30"
        >
          <div
            className="MuiGrid-root MuiGrid-container"
          >
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-45 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-47 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-48 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-49 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-50 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-51 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-52 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-53 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-xl-4"
            >
              <div
                className="MuiPaper-root MuiCard-root makeStyles-cardRoot-37 makeStyles-root-38 MuiPaper-elevation1 MuiPaper-rounded"
              >
                <div
                  className="makeStyles-cornerWrap-42"
                >
                  <div
                    className="makeStyles-iconWrap-44"
                  >
                    <span
                      aria-hidden={true}
                      className="material-icons MuiIcon-root fas fa-users MuiIcon-fontSizeSmall"
                    />
                  </div>
                  <div
                    className="makeStyles-triangle-43"
                  />
                </div>
                <div
                  className="MuiBox-root MuiBox-root-54 makeStyles-mediaWrap-40"
                >
                  <div
                    className="MuiCardMedia-root makeStyles-media-41"
                    style={
                      Object {
                        "backgroundImage": "url(\\"https://picsum.photos/id/412/240/120\\")",
                      }
                    }
                  />
                </div>
                <div
                  className="MuiCardContent-root makeStyles-contentRoot-46"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    募集名が入ります。1 募集名が入ります。1募集名が入ります。1募集名が入ります。
                  </p>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiBox-root MuiBox-root-55"
            slot="container-end"
          >
            <a
              className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-moreLink-31 MuiTypography-colorPrimary"
              href="#"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              <p
                className="MuiTypography-root MuiTypography-body1"
              >
                common:common.see_more
                <span
                  aria-hidden={true}
                  className="material-icons MuiIcon-root makeStyles-moreIcon-32 fas fa-chevron-right MuiIcon-fontSizeSmall"
                />
              </p>
            </a>
          </div>
        </div>
        <div
          className="MuiGrid-root makeStyles-button-33 MuiGrid-item MuiGrid-grid-xs-12"
        >
          <button
            className="MuiButtonBase-root button-primary primary-rounded gradient primary-large"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <div
              className="circle"
            />
            <span
              className="label-primary MuiButton-label"
            >
              common:top.start_exelab
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
]
`;
